package.json

```diff
@@ -150,6 +150,7 @@
         "react-hook-form": "^4.8.2",
         "react-i18next": "^10.12.2",
         "react-router-dom": "^5.0.1",
+        "react-scripts": "^3.4.1",
```

Comment:
Should this not still be a devDependency?

Reply:
You are right. It may have returned as a result of the git cherry-pick from an older branch

Reply:
Actually, just reviewed a new bundle created by CRA and it's actually a dependency and not a dev one. Probably best to stay inline with how react-scripts should be used.